lib/livebook_web/live/home_live.ex

```diff
@@ -320,6 +320,21 @@ defmodule LivebookWeb.HomeLive do
     {:noreply, socket}
   end
 
+  def handle_info({:set_file, file}, socket) do
```

Comment:
By using `/home/open?path=...` you can match on the route in `handle_params` and open the notebook straight away there :)